addons/mail/static/src/js/views/activity/activity_view.js

```diff
@@ -34,6 +35,14 @@ var ActivityView = BasicView.extend({
 
         this.rendererParams.templates = _.findWhere(this.arch.children, { 'tag': 'templates' });
     },
+    /**
+     *
+     * @override
+     */
+    getRenderer(parent, state) {
+        state = Object.assign(state || {}, this.rendererParams);
```

Comment:
wow... it takes me a while to understand how 'arch' was in the initialState of the ActivityController. It's because of this, and it is bad: we write the rendererParams directly in the given state object. You should instead do:
```
state = Object.assign({}, state, this.rendererParams);
```
By the way, can you fix this in PivotView as well? Thanks

When you'll have fixed this, the `this.initialState.arch...` in the controller won't work anymore. You should add the arch in the controllerParams in the init of ActivityView, and store it on `this` in the init of ActivityController.